src/Processor/Braintree/Subscription/Update/BaseUpdateStrategy.php

```diff
@@ -43,6 +43,8 @@ protected function doBraintreeUpdate(Subscription $subscription, ?Plan $newPlan
         $request = Arr::dissoc($request, ['firstBillingDate', 'status']);
         $request['options'] = ['prorateCharges' => true];
 
+        // print_r($request);
```

Comment:
😱

Reply:
:table-flip: - I guess this PR is still wip, lol